File: Coding/coding modul 1/PRAK110-2410817110007-MUHAMMAD-HAMKA-AKBARI.py
import math

print("Diketahui :")
alas = 5
tinggi = 12

print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm\n")

print("jawab :")
print(f"Sisi A = {tinggi} cm")

sisi_miring = math.sqrt(alas**2 + tinggi**2)

print(f"Sisi B = {sisi_miring:.0f} cm")
print(f"Sisi C = {alas} cm")

keliling = alas + tinggi + sisi_miring

print(f"Keliling = {keliling:.0f} cm")

luas = 1/2 * alas * tinggi

print(f"Luas = {luas:.0f} cm")

File: coding modul 1/PRAK110-2410817110007-MUHAMMAD-HAMKA-AKBARI.c
#include <stdio.h>
#include <math.h>

int main()
{
    printf("Diketahui :\n");
    double alas = 5;
    double tinggi = 12;

    printf("Alas = %.0lf cm\n", alas);
    printf("Tinggi = %.0lf cm\n", tinggi);

    printf("\njawab :\n");
    printf("Sisi A = %.0lf cm\n", tinggi);

    double sisi_miring = sqrt(pow(alas, 2) + pow(tinggi, 2));

    printf("Sisi B = %.0lf cm\n", sisi_miring);
    printf("Sisi c = %.0lf cm\n", alas);

    double keliling = alas + tinggi + sisi_miring;

    printf("Keliling = %.0lf cm\n", keliling);

    double luas = 1/2 * alas * tinggi;

    printf("Luas = %.0lf cm");

    return 0;
}